lib/rubocop/cop/rspec/pending.rb

```diff
@@ -40,10 +40,13 @@ class Pending < Base
         def_node_matcher :skippable?, SKIPPABLE.send_pattern
 
         def_node_matcher :skipped_in_metadata?, <<-PATTERN
-          {
-            (send _ _ <#skip_or_pending? ...>)
-            (send _ _ ... (hash <(pair #skip_or_pending? { true str }) ...>))
-          }
+          (send _ _
```

Comment:
maybe we need another type of a helper (to use as a predicate) that takes care of the fact that it might or might not be a hash

Reply:
Indeed. Reverting this meanhile along with the other two.